packages/app/source/client/controllers/todos-controller.js

```diff
@@ -20,23 +23,37 @@ Space.Object.extend(Todos, 'TodosController', {
     this.send(new Todos.CreateTodo({
       targetId: this.configuration.todoListId,
       title: event.title,
-      id: new Guid(),
       isCompleted: false
```

Comment:
great, thought that was the case. Considering we _don't_ have a requirement to add _completed_ items, I think this is the right move